src/controllers/requestController.js

```diff
@@ -0,0 +1,21 @@
+import tripService from '../services/requestService';
+/**
+ * Class for Request
+ */
+export default class RequestController {
+  /**
+   *
+   * @param {object} req - request object
+   * @param {object} res - reponse object
+   * @returns {object} data - trip details
+   */
+  static async returnTrip({ body, userData }, res) {
+    try {
+      const data = await tripService.returnTrip({ ...body, ...userData });
+
+      res.status(201).json({ status: 'success', data });
```

Comment:
there is a helper function for responses in the utils folder please utilise it

Reply:
please consider refactoring the response class
![Screen Shot 2019-09-01 at 2 31 44 PM](https://user-images.githubusercontent.com/46590803/64077803-aaf32680-cccb-11e9-881e-358158a1d5cb.png)